grails-app/taglib/images/client/plugin/ImageClientTagLib.groovy

```diff
@@ -1,27 +1,30 @@
 package images.client.plugin
 
-import au.org.ala.web.AuthService
+import grails.config.Config
+import grails.core.support.GrailsConfigurationAware
 
 /**
  * Image Client Tag Lib to be used in the gsp related to image client plugin feature
  *
  */
-class ImageClientTagLib {
-
-    AuthService authService
+class ImageClientTagLib implements GrailsConfigurationAware {
     static namespace = 'imageClient'
 
-    /**
+    List<String> allowedRoles = []
+
+    @Override
+    void setConfiguration(Config config) {
+        def roleList = config.getProperty("allowedImageEditingRoles", "")
+        allowedRoles = roleList ? roleList.split(",") : []
```

Comment:
Minor thing - is it worth cleaning up any whitespace from the roleList given it can be hand edited in external configuration ?

Reply:
Fixed in https://github.com/AtlasOfLivingAustralia/images-client-plugin/pull/17